packages/preview-middleware-client/tsconfig.eslint.json

```diff
@@ -1,10 +1,7 @@
 {
     "extends": "./tsconfig.json",
-    "include": [".eslintrc.js", "src", "test", "types"],
+    "include": [".eslintrc.js", "jest.config.js", "src", "test", "types"],
     "compilerOptions": {
-        "types": [
-            "@sapui5/types",
-            "jest"
-        ]
+        "types": ["@sapui5/types", "node", "jest"]
```

Comment:
Why do we need `node` here? This is all client-side coding, right?

Reply:
The config is for both client side coding and tests, which run in node. Do you think we need separate tsconfig files for each environment?

Reply:
No need to overengineer it. I was wondering why it wasn't required before, and I also naively just deleted it and ran `pnpm lint` without issues. That's why I was wondering why it is needed.

Reply:
Review this as part of #1328